test/utils/browser.js

```diff
@@ -1,23 +1,45 @@
 // Load in dependencies
 var assert = require('assert');
+var asserters = require('wd/lib/asserters');
+var async = require('async');
 var fs = require('fs');
 var functionToString = require('function-to-string');
-var async = require('async');
 var wd = require('wd');
```

Comment:
I think I fucked up the order with `async` but the order in this repo should be:

1. First party modules, alphabetical by `var` (e.g. `assert`, `fs`)
2. Third party modules, alphabetical by `var` (e.g. `function-to-string`, `wd`)
3. Local modules, alphabetical by var (e.g. `./utils/browser`)